2023-07-26 18:45:31.709 UTC [2142737] LOG:  starting PostgreSQL 15.3 on x86_64-pc-linux-gnu, compiled by gcc (Ubuntu 11.3.0-1ubuntu1~22.04.1) 11.3.0, 64-bit
2023-07-26 18:45:31.709 UTC [2142737] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2023-07-26 18:45:31.724 UTC [2142737] LOG:  listening on Unix socket "/tmp/.s.PGSQL.5432"
2023-07-26 18:45:31.738 UTC [2142740] LOG:  database system was shut down at 2023-07-26 18:45:31 UTC
2023-07-26 18:45:31.747 UTC [2142737] LOG:  database system is ready to accept connections
2023-07-26 18:45:31.976 UTC [2142756] ERROR:  large object 43213 does not exist
2023-07-26 18:45:31.976 UTC [2142756] STATEMENT:  SELECT lo_get(43213);
2023-07-26 18:45:31.983 UTC [2142756] ERROR:  large object 43214 does not exist
2023-07-26 18:45:31.983 UTC [2142756] STATEMENT:  SELECT lo_get(43214);
2023-07-26 18:45:32.002 UTC [2142738] LOG:  checkpoint starting: immediate force wait
2023-07-26 18:45:32.071 UTC [2142738] LOG:  checkpoint complete: wrote 9 buffers (0.1%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.020 s, sync=0.014 s, total=0.069 s; sync files=7, longest=0.008 s, average=0.002 s; distance=4081 kB, estimate=4081 kB
2023-07-26 18:45:32.612 UTC [2142738] LOG:  checkpoint starting: immediate force wait
2023-07-26 18:45:32.763 UTC [2142738] LOG:  checkpoint complete: wrote 1793 buffers (10.9%); 0 WAL file(s) added, 0 removed, 1 recycled; write=0.064 s, sync=0.047 s, total=0.151 s; sync files=496, longest=0.015 s, average=0.001 s; distance=12312 kB, estimate=12312 kB
2023-07-26 18:45:32.788 UTC [2142738] LOG:  checkpoint starting: immediate force wait
2023-07-26 18:45:32.844 UTC [2142738] LOG:  checkpoint complete: wrote 2 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.012 s, sync=0.010 s, total=0.056 s; sync files=2, longest=0.007 s, average=0.005 s; distance=1 kB, estimate=11081 kB
2023-07-26 18:45:32.972 UTC [2142796] ERROR:  current user cannot be dropped
2023-07-26 18:45:32.972 UTC [2142796] STATEMENT:  DROP USER su;
2023-07-26 18:45:33.426 UTC [2142738] LOG:  checkpoint starting: immediate force wait
2023-07-26 18:45:33.560 UTC [2142738] LOG:  checkpoint complete: wrote 1017 buffers (6.2%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.064 s, sync=0.037 s, total=0.134 s; sync files=297, longest=0.010 s, average=0.001 s; distance=8832 kB, estimate=10856 kB
2023-07-26 18:45:33.620 UTC [2142738] LOG:  checkpoint starting: immediate force wait
2023-07-26 18:45:33.677 UTC [2142738] LOG:  checkpoint complete: wrote 8 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.012 s, sync=0.011 s, total=0.058 s; sync files=7, longest=0.008 s, average=0.002 s; distance=34 kB, estimate=9774 kB
2023-07-26 18:45:33.792 UTC [2142824] ERROR:  duplicate key value violates unique constraint "company_pkey"
2023-07-26 18:45:33.792 UTC [2142824] DETAIL:  Key (id)=(1) already exists.
2023-07-26 18:45:33.792 UTC [2142824] STATEMENT:  INSERT  INTO Company(ID, Name) VALUES (1, 'Percona');
2023-07-26 18:45:34.056 UTC [2142844] ERROR:  division by zero
2023-07-26 18:45:34.056 UTC [2142844] STATEMENT:  SELECT 1/0;
2023-07-26 18:45:34.057 UTC [2142844] ERROR:  relation "unknown" does not exist at character 15
2023-07-26 18:45:34.057 UTC [2142844] STATEMENT:  SELECT * FROM unknown;
2023-07-26 18:45:34.057 UTC [2142844] ERROR:  syntax error at or near "ELECET" at character 1
2023-07-26 18:45:34.057 UTC [2142844] STATEMENT:  ELECET * FROM unknown;
2023-07-26 18:45:34.057 UTC [2142844] WARNING:  warning message
2023-07-26 18:45:34.057 UTC [2142844] CONTEXT:  PL/pgSQL function inline_code_block line 3 at RAISE
2023-07-26 18:45:34.191 UTC [2142858] ERROR:  role "su" already exists
2023-07-26 18:45:34.191 UTC [2142858] STATEMENT:  CREATE USER su WITH SUPERUSER;
2023-07-26 18:45:34.219 UTC [2142858] ERROR:  permission denied for function pg_stat_monitor_reset
2023-07-26 18:45:34.219 UTC [2142858] STATEMENT:  SELECT pg_stat_monitor_reset();
